docs/source/AdministratorGuide/index.rst

```diff
@@ -14,6 +14,7 @@ This administration documentation refers to the "Core" DIRAC project.
    :maxdepth: 2
 
    Introduction/index
+   ExternalsSupport/index
```

Comment:
I think it would be better to make this a part of the ServerInstallations section, instead of adding another top level heading here https://dirac.readthedocs.io/en/latest/AdministratorGuide/index.html

Reply:
I disagree, as not everything goes and that will go in this section is for server installation.